org.flameshot.flameshot.yml

```diff
@@ -0,0 +1,33 @@
+app-id: org.flameshot.flameshot
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: flameshot
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  - --device=dri
+  # Connectivity
+  - --share=network
+  # QtSingleApplication, allow other instances to see log files
+  - --env=TMPDIR=/var/tmp
+  # Allow loading/saving files from anywhere
+  - --filesystem=host
+  # Notification access
+  - --talk-name=org.freedesktop.Notifications
+  # System Tray Icon
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=org.kde.StatusNotifierItem-2-1
+modules:
+  - name: flameshot
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: git
+        url: https://github.com/flameshot-org/flameshot.git
+        branch: "flathub"
+        commit: "700ad3cf7361aa9077d94c978e7aad6a97faca84"
```

Comment:
This should be a stable release unless this one is

Reply:
Yeah I will move to our master branch and use tag / commit rather than branch / commit. I was doing this work on a side branch to get everything working before merging.

Reply:
Sounds good, let us know once it's ready